python/requirements/orca/requirement_pyspark.txt

```diff
@@ -0,0 +1 @@
+2.4.6
```

Comment:
Can we change to pyspark==2.4.6 and grep the version out?

Reply:
Any "pyspark==2.4.6" will be scanned as security issue. We need to avoid the key word "pyspark==".

Reply:
@hkvision @leonardozcm Another idea. How about write PYSPARK_VERSION='2.4.6' in dllib setup.py directly instead of putting it in requirement.txt?

Reply:
> @hkvision @leonardozcm Another idea. How about write PYSPARK_VERSION='2.4.6' in dllib setup.py directly instead of putting it in requirement.txt?

I am ok with this one

Reply:
> I am ok with this one

I think it is a good idea